pyproject.toml

```diff
@@ -49,15 +49,94 @@ line_length = 100
 combine_as_imports = true
 filter_files = true
 
+[tool.ruff]
+line-length = 100
+target-version = "py38"
+src = ["disnake"]
```

Comment:
```suggestion
```
Probably doesn't matter, however `src` defaults to `["."]`, which is what we want.
https://beta.ruff.rs/docs/settings/#src

Reply:
https://github.com/DisnakeDev/disnake/pull/935/commits/2a9430fb19dd528de7d1cca5667b561a4f9ad195